tests/test_update.py

```diff
@@ -611,6 +612,13 @@ def test_signed_updates(
         """Test various combinations of signed and unsigned, present and non-
         present verification keys."""
 
+        if sig_case.artifact_version == 2 and version_is_minimum(
+            bitbake_variables, "mender", "4.0.0"
+        ):
+            pytest.skip(
```

Comment:
Q: Why not skipif?

Reply:
Skipif doesn't support using fixture variables in the expression, and I need `bitbake_variables`.